UnitsNet.Tests/CustomCode/StonePoundsTests.cs

```diff
@@ -35,9 +35,9 @@ public void StonePoundsToString_FormatsNumberInDefaultCulture()
         {
             Mass m = Mass.FromStonePounds(3500, 1);
             StonePounds stonePounds = m.StonePounds;
-            string numberInCurrentCulture =  3500.ToString("n0", CultureInfo.CurrentCulture); // Varies between machines, can't hard code it
+            string numberInCurrentCulture = 3500.ToString("n0", CultureInfo.InvariantCulture);
```

Comment:
The test name indicates it tests the default culture, meaning ToString() without explicitly passing a culture.

For these kinds of tests, we could just set `CultureInfo.CurrentCulture` instead. 
As far as I know, manipulating `CurrentCulture` in one test case does not affect other test cases due to how the test runner works. I might be wrong.

I'll comment similar instances in other places with `TODO Set CurrentCulture instead of passing culture argument to ToString()`.

Reply:
Disagree with the remark, except that the test title is incorrect.

If this test is checking for the Default culture, which is the Current one, then why does Assert.Equal use the English language?

The result will always be false for those cultures that have unit localizations.

Reply:
This is what I propose:

```cs
        [Fact]
        public void StonePoundsToString_FormatsNumberInCurrentCulture()
        {
            CultureInfo.CurrentCulture = CultureInfo.GetCultureInfo("en-US");
            StonePounds stonePounds = Mass.FromStonePounds(3500, 1).StonePounds;

            Assert.Equal("3,500 st 1 lb", stonePounds.ToString());
        }
```

Rename it to `InCurrentCulture`, set the current culture at the top and simply call `ToString()` without any culture argument. Other tests in this test class test the behavior of passing an explicit culture, so it makes sense to let this test the implicit culture.

Does that make sense?

Reply:
I understand the difference, but the check will be the same.
Either we are explicitly specifying InvariantCulture, either we are explicitly specifying the English culture.
Anyway, I did as you said.